fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java

```diff
@@ -338,6 +341,14 @@ public Type mysqlTypeToDoris(JdbcFieldSchema fieldSchema) {
                     return Type.BIGINT;
                 case "BIGINT":
                     return ScalarType.createStringType();
+                case "DECIMAL":
```

Comment:
Please add regression test for this case

Reply:
done